sources/Context.cpp

```diff
@@ -211,12 +211,12 @@ Context::exportOutputJob() {
   switch (def_.simulationKind) {
   case SimulationKind::SECURITY_ANALYSIS:
     // For security analysis always export the main jobs file, as dynawo-algorithms will need it
-    outputs::Job::exportJob(jobEntry_, absolute(def_.networkFilepath), config_.outputDir());
+    jobWriter.exportJob(jobEntry_, absolute(def_.networkFilepath), config_.outputDir());
```

Comment:
as it is static please call through the class name

Reply:
Done